ocaml/idl/datamodel.ml

```diff
@@ -641,6 +641,10 @@ let _ =
     ~doc:"VGPU type is not one of the PGPU's supported types." ();
   error Api_errors.vgpu_type_not_compatible_with_running_type ["pgpu"; "type"; "running_type"]
     ~doc:"VGPU type is not compatible with one or more of the VGPU types currently running on this PGPU" ();
+  error Api_errors.vgpu_not_compatible_with_destination_pgpu ["reason"; "vgpu"; "pgpu"]
+    ~doc:"The VGPU is not compatible with any of the destination PGPU." ();
```

Comment:
Yes... there is an inconsistency when we use this error for `asset_can_migrate` (in the future) and in `pool_migrate`. In one case we test all (although they are supposed to be equal) and in the other we check only the allocated one.